src/types.js

```diff
@@ -85,3 +85,8 @@ export type Recipient = {
   subject: string,
   email: string,
 };
+
+export type ApiResponse = {
```

Comment:
We can add a TODO and do this in another PR as well to get things merged quickly :)